### PR TITLE
Creates tests and improve Debug command

### LINKED ISSUE #11: Create tests + refactor 'debug' command

### PR DESCRIPTION
**Pull request by Hikari-desuyoo**
## Solution
- Created new ENV value that tells which debugger the user will be using
- Changed ENV behaviour to use values from template .env if said value was not found on the main .env
- Changes default ENV values in favor of more popular ones

## Related links
- closes #11

### CODE REVIEW

File: spec/commands/debug_spec.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  it 'Calls debug if debugger is debug' do
+    ENV['DEBUGGER'] = 'debug'
+    command_obj = Commands::Debug.new(ArgsHandler.parse('debug'))
+    command_binding = double('binding', break: nil)
+    allow(command_obj).to receive(:binding) { command_binding }
+    command_obj.execute
+    expect(command_binding).to have_received(:break)
+  end
+end
```
Comment (molvrr) [inline spec/commands/debug_spec.rb:29]: What do you think about separating each test in act, arrange and assert so we can improve readability?

File: utils/dependencies.rb [hunk 2]
```diff
@@ -45,4 +45,6 @@ def load_loader(loader_path)
 Dir[File.dirname(__FILE__) + '/../utils/**/base.rb'].each { |file| require_relative file }
 Dir[File.dirname(__FILE__) + '/../utils/**/*.rb'].each { |file| require_relative file }
 load_loaders
+
+Dotenv.overload('templates/.env.example')
 Dotenv.overload
```
Comment (molvrr) [inline utils/dependencies.rb:50]: We can just pass `templates/.env.example` and `.env` as arguments for `Dotenv.overload`, like this: `Dotenv.overload('.env', 'templates/.env.example')`. The values from the first argument will have precedence

**Review by molvrr — CHANGES_REQUESTED** [review]

File: utils/dependencies.rb (hunk 2 above)
Comment (Hikari-desuyoo) [reply utils/dependencies.rb]: I made it work here, but the second argument seems to have precedence instead.

File: spec/commands/debug_spec.rb (hunk 1 above)
Comment (Hikari-desuyoo) [reply spec/commands/debug_spec.rb]: Good idea, thanks for your review!

File: utils/dependencies.rb (hunk 2 above)
Comment (molvrr) [reply utils/dependencies.rb]: :rock: :rocket: :100:

**Pull request merged**